src/Tokenizer/Analyzer/NamespaceUsesAnalyzer.php

```diff
@@ -203,8 +203,11 @@ private function getNearestQualifiedName(Tokens $tokens, int $index): array
             $index = $tokens->getNextMeaningfulToken($index);
         }
 
+        /** @var class-string $fqn */
+        $fqn = $fullName;
```

Comment:
I had to do that because of PHPStan, which expects `class-string`, but infers `string` from the code above (rightfully), and I couldn't do `/** @var class-string $fullName */` before `return` because Fixer wants to change it to `//` comment and then PHPStan can't see the enforced type 😕.